contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -693,8 +691,7 @@ var ThumbnailUploadView = BaseViews.BaseView.extend({
     },
     image_completed:function(){
         if(this.image_error){
-            var self = this;
-            dialog.dialog("Image Error", this.image_error, { "OK":function(){} }, null);
+            dialog.alert("Image Error", this.image_error, null);
```

Comment:
Seems like the callback parameter is being used about as often as not. Might be worth setting a default value? Checking to see if it's valid, and assigning it if it isn't.